jmx-metrics/src/main/resources/target-systems/cassandra.groovy

```diff
@@ -30,7 +30,7 @@ otel.instrument(clientRequestRangeSliceLatency,
 
 otel.instrument(clientRequestRangeSliceLatency,
         "cassandra.client.request.range_slice.latency.count",
-        "Total token range read request latency", "µs", "Count",
+        "Total token range read request latency", "1", "Count",
```

Comment:
Is it a good time to update the description too? Seems to not be accurate - I'd expect "total latency" to be temporal since it's still latency